readme.md

```diff
@@ -69,6 +69,7 @@ Click the type names for complete docs.
 - [`Merge`](source/merge.d.ts) - Merge two types into a new type. Keys of the second type overrides keys of the first type.
 - [`MergeExclusive`](source/merge-exclusive.d.ts) - Create a type that has mutually exclusive properties.
 - [`RequireAtLeastOne`](source/require-at-least-one.d.ts) - Create a type that requires at least one of the given properties.
+- [`RequireOne`](source/require-one.d.ts) - Create a type that requires a single property of the given properties, but not more.
```

Comment:
The description should be clearer about it only allowing one of the properties. Regarding, https://github.com/sindresorhus/type-fest/pull/55/files#r310352159